### DISCUSSION TITLE
After deploy: no prod.log

### DISCUSSION THREAD

**Discussion by Adi-18** [symfony/symfony#45146] [Q&A] [answered]
### Symfony version(s) affected

6.0.0

### Description

I installed new environment with Symfony 6.0.1 and it works on dev.

When I copy to prod environment and set **APP_ENV=dev** and **APP_DEBUG=false** everything works except one route. I got The server returned a "500 Internal Server Error".

But I don't get prod.log (no prod.log file, permission is ok).

When I set **APP_DEBUG=true** the related route works without exception.

### How to reproduce

Installed new Symfony 6 project and copied sources from old project 4.
Error "500 Internal Server Error" was because I used
{% if false %}
Test: **_{{ dump(viewtest) }}_**
{%  endif %}

**Comment by Adi-18** [comment]
Fresh installation on independent System. There also no prod.log.
 Also not, if I run
 
    /**
     * @Route(
     *     "/test-log",
     *     name="test-log",
     *     methods={"GET","POST"}
     * )
     * @param LoggerInterface $logger
     * @return Response
     */
    public function testLog(LoggerInterface $logger): Response
    {
           $logger->info('I just got the logger');
            return $this->render('main/index.html.twig', []);
    }

**Comment by fd6130** [comment, marked as answer]
I think because they have change the recipes of monolog, go to `/config/prod/monolog.yml` and locate `nested`
```
nested:
            type: stream
            path: php://stderr
            level: debug
            formatter: monolog.formatter.json
```
Change `path: php://stderr` to `path: "%kernel.logs_dir%/%kernel.environment%.log"` and you are good to go.

**Reply by Adi-18** [reply]
Yes, that's the point. Thank you
